set.mm

```diff
@@ -433649,8 +433651,8 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
 htmldef "^o" as
     " <IMG SRC='_hato.gif' WIDTH=11 HEIGHT=19 ALT=' ^o' TITLE='^o'> ";
   althtmldef "^o" as ' &uarr;<SUB>&#x1D45C;</SUB> ';
-  latexdef "^o" as "\uparrow_o"; /*
-  latexdef "^o" as "\hat{\ }_o"; */
+  latexdef "^o" as "\uparrow_o";
+    /* latexdef "^o" as "\hat{\ }_o"; */
```

Comment:
```suggestion
```
no need to keep the old definition

Reply:
I agree, if this general policy is valid for all latexdefs, I might do a pull request to remove them.

Reply:
Maybe not general, since sometimes it's a matter of choice. Here, the present definition is clearly better than the old one.

Reply:
ok